### PR TITLE
[bug 1127312] Document how to keep up with changes to fjord in the dev environment

### PR DESCRIPTION
**Pull request by lgp171188**
Explain the common problem scenarios encountered on pulling from
Mozilla's repository and how to resolve them and keep the fjord
development environment up to date.

Please review the content and suggest corrections if any.

### CODE REVIEW

File: docs/getting_started.rst [hunk 1]
```diff
@@ -432,6 +432,38 @@ timezone on the host and the VM, run::
...
+keep up with the new changes, here are some tips.
+
+- Use the appropriate git commands to update the fjord repository
+  depending on which git workflow you use. For example, if you are
+  tracking the main repository on the master branch of your fork,
+  you will have to run ``git pull`` when on the master branch.
+- If there are database changes in the updates to the repository, those
+  have to be applied. See the section on ``migrate`` command above.
+- If any of the fjord dependencies have changed, you will see an error
+  message like the one below when you run any ``manage.py`` command
```
Comment (willkg) [inline docs/getting_started.rst]: We probably want to make the next block a "code sample". You do that by adding `::` to the end of this line.

File: docs/getting_started.rst [hunk 2]
```diff
@@ -432,6 +432,38 @@ timezone on the host and the VM, run::
...
 and select your current timezone as the timezone for the VM.
 
+Keeping up with changes to Fjord
+================================
+
+Fjord is in active development and there may be changes made everyday.
+So if you have an existing fjord development environment and want to
+keep up with the new changes, here are some tips.
+
+- Use the appropriate git commands to update the fjord repository
```
Comment (willkg) [inline docs/getting_started.rst]: Nit: I've been using `*` for bullets.

File: docs/getting_started.rst [hunk 3]
```diff
@@ -432,6 +432,38 @@ timezone on the host and the VM, run::
...
+  In such a scenario, you have to find out the requirements files in which
+  the unsatisfied requirements are listed and the use ``peep`` to install them.
+  For example you might have to run
+
+    ./peep install -r requirements/dev.txt
+
+  if there are unsatisfied requirements in ``requirements/dev.txt``.
+- There might be updates to the product details and to get them, run
+  the ``update_product_details`` command using ``manage.py`` as described
+  above.
```
Comment (willkg) [inline docs/getting_started.rst]: The product details are maintained in an svn repository elsewhere. There are always updates in there, but those updates aren't related to fjord development. i.e. We can make all the changes we want, but none of them affect the product details svn repository. Ergo, I think for now I'd leave this section out.

**Comment by willkg** [comment]
I think this looks good. If you make those changes, I'll merge this in and then probably write an additional chapter on maintaining requirements that parallels the maintaining vendor chapter which will go away at some point.

**Comment by lgp171188** [comment]
@willkg I have made the suggested changes. Please let me know if there is anything else to fix.

**Comment by willkg** [comment]
Looks good! Thank you!

**Comment by willkg** [comment]
2c76974 [bug 1127312] Document how to keep up with changes to fjord

**Pull request merged**